### ISSUE TITLE
doesn't work

### ISSUE THREAD

**Issue by hemna** [cafehaine/xontrib-xlsd#31]
```╰─ $ xontrib load xlsd
xonsh: For full traceback set: $XONSH_SHOW_TRACEBACK = True
The xonsh.proc module has been deprecated in favor of the xonsh.procs subpackage.
╭─i530566(xonsh) at C17ZL21PMD6M in ~/.dotfiles
╰─ $ ls
xonsh: For full traceback set: $XONSH_SHOW_TRACEBACK = True
AttributeError: module 'xonsh.platform' has no attribute 'scandir'
xonsh: For full traceback set: $XONSH_SHOW_TRACEBACK = True
xonsh.proc.STDOUT_CAPTURE_KINDS has been deprecated. please use xonsh.procs.pipelines.STDOUT_CAPTURE_KINDS instead.
```

```
╭─i530566(xonsh) at C17ZL21PMD6M in ~/.dotfiles
╰─ $ xonfig
+------------------+----------------------+
| xonsh            | 0.9.27               |
| Git SHA          | 71fe9014             |
| Commit Date      | Jan 29 08:58:58 2021 |
| Python           | 3.8.7                |
| PLY              | 3.11                 |
| have readline    | True                 |
| prompt toolkit   | 3.0.16               |
| shell type       | prompt_toolkit       |
| pygments         | 2.8.0                |
| on posix         | True                 |
| on linux         | False                |
| on darwin        | True                 |
| on windows       | False                |
| on cygwin        | False                |
| on msys2         | False                |
| is superuser     | False                |
| default encoding | utf-8                |
| xonsh encoding   | utf-8                |
| encoding errors  | surrogateescape      |
| on jupyter       | False                |
| jupyter kernel   | None                 |
| xontrib 1        | abbrevs              |
| xontrib 2        | argcomplete          |
| xontrib 3        | autovox              |
| xontrib 4        | autoxsh              |
| xontrib 5        | avox                 |
| xontrib 6        | avox_poetry          |
| xontrib 7        | back2dir             |
| xontrib 8        | cmd_done             |
| xontrib 9        | commands             |
| xontrib 10       | coreutils            |
| xontrib 11       | direnv               |
| xontrib 12       | docker_tabcomplete   |
| xontrib 13       | fzf-widgets          |
| xontrib 14       | gitinfo              |
| xontrib 15       | histcpy              |
| xontrib 16       | vox                  |
| xontrib 17       | voxapi               |
| xontrib 18       | xlsd                 |
| xontrib 19       | z                    |
+------------------+----------------------+
```

**Comment by cafehaine** [comment]
It is actually fixed in master, I'll update the changelog and create a new release to fix this soon

**Comment by cafehaine** [comment]
In the meantime you can install the master version of xlsd using `pip install git+https://github.com/cafehaine/xontrib-xlsd`

**Comment by cafehaine** [comment]
The issue you were having should be fixed, however you still won't be able to use xlsd as you are running python3.8, but xlsd requires 3.9.

**Comment by diego-rapoport** [comment]
> In the meantime you can install the master version of xlsd using `pip install git+https://github.com/cafehaine/xontrib-xlsd`

That worked perfectly for me!

**Comment by djmattyg007** [comment]
I'd really appreciate it if you could create a new release that includes this fix :pray:

**Comment by cafehaine** [comment]
A new release is now available (v0.1.5) which should fix the scandir issue. As the original post was for an unrelated issue which should be fixed by now, I'm closing this thread.

Please open another issue if you encounter any problem.
